manifests/08_remover_job.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: openshift-service-catalog-controller-manager-remover
+  namespace: openshift-service-catalog-controller-manager-remover
+spec:
+  template:
+    spec:
+      containers:
+      - name: pi
+        # image will be some registry image
+        image: registry.svc.ci.openshift.org/openshift/origin-v4.0:cluster-svcat-controller-manager-operator
```

Will the image name still be `registry.svc.ci.openshift.org/openshift/origin-v4.0:cluster-svcat-controller-manager-operator` or should it be:

```suggestion
        image: registry.svc.ci.openshift.org/openshift/origin-v4.0:cluster-svcat-controller-manager-remover
```

---

@joelanford my understanding was we're keeping the image name the same, mostly because it is less work since otherwise we'd have to add a bunch of supporting stuff to add a new image to the payload. And it made sense for the operator images to be there to remove their old selves from the cluster.